webview/features/Sidebar/Collection/SidebarCollection.js

```diff
@@ -142,7 +142,8 @@ const UtilitySectionWrapper = styled.div`
     margin-top: 1rem;
     padding: 0.4rem 0.6rem;
     border-radius: 0.35rem;
-    color: white;
+    color: var(--default-text);
+    opacity: 0.78;
   }
 `;
 
```

Comment:
All changes on this PR look nice, but there still seems to be an issue with the white VSCode theme. 
Although it's not as popular as darker themes you wanna try solving this problem(Could be annoying to fix this)? 

If you are satisfied with the current change and don't wanna bother dealing with the white VSCode theme, it's totally fine!

Reply:
ah yes i never tested it out with white i will take a look later today.

Reply:
Awesome! Thanks!!

Reply:
Hi i never managed to find a way to fix it unfortunately so i say merge it and in the future if someone raises the issue we can take a better look

Reply:
Sounds good!